clients/wallet/src/keys.rs

```diff
@@ -0,0 +1,36 @@
+#[allow(unused_imports)]
+use dotenv::dotenv;
+use std::env;
+// Gets env variables with precedence from the system environment and then from the .env file.
+// If one variable is not on the system environment, and the .env is defined, then all variables 
+// will be overridden by those on the .env file.
```

Comment:
```suggestion
use std::env;

/// Gets env variables with precedence from the system environment and then from the .env file.
/// If one variable is not on the system environment, and the .env is defined, then all variables 
/// will be overridden by those on the .env file.
```

Reply:
> If one variable is not on the system environment, and the .env is defined, then all variables will be overridden by those on the .env file.

I'm not sure I understand how this works. Is it because the call to `dotenv::from_filename()` will overwrite all env variables and this is only called once the first value is not found from the system environment?

Reply:
That was the idea. But I have modified so the .env takes precedence, and only if one variable is not declared there but has been passed to the environment it can also be used.